### PR TITLE
Delete dead code

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Fixes https://github.com/dotnet/roslyn/issues/36138

### CODE REVIEW

**Review by jasonmalinowski — APPROVED** [review]
Fine to delete, although I kinda wonder if we want this... :-/

**Comment by CyrusNajmabadi** [comment]
Haven't needed it in 4 years.  So either it goes, or someone makes a PR for it to come back.  Regardless, it's in the history.  So if it's needed just reference that :)

File: src/VisualStudio/Core/Def/ProjectSystem/Legacy/AbstractLegacyProject_IIntellisenseBuildTarget.cs [hunk 1]
```diff
@@ -13,68 +11,5 @@ internal partial class AbstractLegacyProject : IIntellisenseBuildTarget
...
-                title: ServicesVSResources.Project_loading_failed,
-                description: GetDescription(reason),
-                helpLink: "http://go.microsoft.com/fwlink/p/?LinkID=734719");
-        }
-
-        private string GetDescription(string reason)
-        {
-            var logFilePath = $"{Path.GetTempPath()}\\{Path.GetFileNameWithoutExtension(this.VisualStudioProject.FilePath)}_*.designtime.log";
-
-            var logFileDescription = string.Format(ServicesVSResources.To_see_what_caused_the_issue_please_try_below_1_Close_Visual_Studio_long_paragraph_follows, logFilePath);
```
Comment (sharwell) [inline src/VisualStudio/Core/Def/ProjectSystem/Legacy/AbstractLegacyProject_IIntellisenseBuildTarget.cs:70]: ❔ Do any of these strings exist?

Comment (CyrusNajmabadi) [reply src/VisualStudio/Core/Def/ProjectSystem/Legacy/AbstractLegacyProject_IIntellisenseBuildTarget.cs:70]: looks like yes.  i'll remove them as well.

**Pull request merged**